notifications/bridge.go

```diff
@@ -108,7 +109,12 @@ func (b *bridge) createManifestEvent(action string, repo string, sm distribution
 	event.Target.Size = int64(len(p))
 	event.Target.Digest = digest.FromBytes(p)
 
-	event.Target.URL, err = b.ub.BuildManifestURL(repo, event.Target.Digest.String())
+	ref, err := reference.WithDigest(repo, event.Target.Digest)
```

Comment:
Not related to this change per se, but can `WithDigest` actually error?  I see digest is matched against a regex in the reference package, but at that point it already appears to be a valid digest object.